static/app/api/urlpatterns.generated.ts

```diff
@@ -0,0 +1,794 @@
+  '/organizations/$organizationIdOrSlug/dashboards/$dashboardId/favorite/',
+  '/organizations/$organizationIdOrSlug/dashboards/$dashboardId/visit/',
+  '/organizations/$organizationIdOrSlug/dashboards/starred/',
+  '/organizations/$organizationIdOrSlug/dashboards/starred/order/',
+  '/organizations/$organizationIdOrSlug/dashboards/widgets/',
+  '/organizations/$organizationIdOrSlug/data-conditions/',
+  '/organizations/$organizationIdOrSlug/data-export/',
+  '/organizations/$organizationIdOrSlug/data-export/$dataExportId/',
+  '/organizations/$organizationIdOrSlug/data-scrubbing-selector-suggestions/',
+  '/organizations/$organizationIdOrSlug/derive-code-mappings/',
+  '/organizations/$organizationIdOrSlug/detector-types/',
+  '/organizations/$organizationIdOrSlug/detector-workflow/',
+  '/organizations/$organizationIdOrSlug/detector-workflow/$detectorWorkflowId/',
+  '/organizations/$organizationIdOrSlug/detectors/',
+  '/organizations/$organizationIdOrSlug/detectors/$detectorId/',
+  '/organizations/$organizationIdOrSlug/detectors/count/',
+  '/organizations/$organizationIdOrSlug/discover/homepage/',
+  '/organizations/$organizationIdOrSlug/discover/saved/',
+  '/organizations/$organizationIdOrSlug/discover/saved/$queryId/',
+  '/organizations/$organizationIdOrSlug/discover/saved/$queryId/visit/',
+  '/organizations/$organizationIdOrSlug/dynamic-sampling/custom-rules/',
+  '/organizations/$organizationIdOrSlug/environments/',
+  '/organizations/$organizationIdOrSlug/eventids/$eventId/',
+  '/organizations/$organizationIdOrSlug/events-facets-performance-histogram/',
+  '/organizations/$organizationIdOrSlug/events-facets-performance/',
+  '/organizations/$organizationIdOrSlug/events-facets/',
+  '/organizations/$organizationIdOrSlug/events-has-measurements/',
+  '/organizations/$organizationIdOrSlug/events-histogram/',
+  '/organizations/$organizationIdOrSlug/events-meta/',
+  '/organizations/$organizationIdOrSlug/events-root-cause-analysis/',
+  '/organizations/$organizationIdOrSlug/events-span-ops/',
+  '/organizations/$organizationIdOrSlug/events-spans-histogram/',
+  '/organizations/$organizationIdOrSlug/events-spans-performance/',
+  '/organizations/$organizationIdOrSlug/events-spans-stats/',
+  '/organizations/$organizationIdOrSlug/events-spans/',
+  '/organizations/$organizationIdOrSlug/events-stats/',
+  '/organizations/$organizationIdOrSlug/events-timeseries/',
+  '/organizations/$organizationIdOrSlug/events-trace-light/$traceId/',
+  '/organizations/$organizationIdOrSlug/events-trace-meta/$traceId/',
+  '/organizations/$organizationIdOrSlug/events-trace/$traceId/',
+  '/organizations/$organizationIdOrSlug/events-trends-stats/',
+  '/organizations/$organizationIdOrSlug/events-trends-statsv2/',
+  '/organizations/$organizationIdOrSlug/events-trends/',
+  '/organizations/$organizationIdOrSlug/events-vitals/',
+  '/organizations/$organizationIdOrSlug/events/',
+  '/organizations/$organizationIdOrSlug/events/$projectIdOrSlug:$eventId/',
```

Comment:
this one is troublesome. It’s why added [colon-splitting](https://github.com/getsentry/sentry/pull/97155/files#diff-4ca7f7c2ece579501ed5679e16c09a64bad48d9226bd1b5148790274d5a8e640R5) and [an extra test](https://github.com/getsentry/sentry/pull/97155/files#diff-f0ef890aa6ca2753302ba96aa070514589cf387b4b78661c7de51baf42a854d7R35-R48). Feel free to re-use that!